examples/react_webpack/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "jsx": "react",
+        "lib": ["ES2015", "DOM"],
+        "listEmittedFiles": true,
+        "traceResolution": true
```

Comment:
Do these two options need to be enabled?

Reply:
no, these are for debugging.
I think maybe the rule implementation could enable these when running with VERBOSE_LOGS in the environment...